server/src/option.rs

```diff
@@ -72,6 +73,22 @@ impl Config {
         }
     }
 
+    pub async fn validate_storage(&self, storage: &impl ObjectStorage) {
+        match storage.check().await {
+            Ok(_) => (),
+            Err(ObjectStorageError::NoSuchBucket(name)) => panic!(
+                "Could not start because the bucket named {bucket} doesn't exist. Please make sure bucket with the name {bucket} exists on {url} and then start parseable again",
+                bucket = name,
+                url = self.storage.endpoint_url()
```

Comment:
This is one of the usage for the error kind which fixes #42. Panic when bucket does not exist or we fail to connect to storage endpoint

Are these error message good ?

Reply:
checking